### PR TITLE
net-im/qtox: revbump with equal slot added to dev-qt/qtcore dep to tr…

### PR DESCRIPTION
**Pull request by soredake**
…igger rebuild automatically when qt slot changes

### CODE REVIEW

File: net-im/qtox/qtox-1.7.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+IUSE="gtk X"
+
+# needed, since tarball provided by github extracts to `qTox`
+S="${WORKDIR}/qTox-${PV}"
+
+RDEPEND="
+	dev-db/sqlcipher
+	dev-libs/libsodium
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5=
```
Comment (gktrk) [inline net-im/qtox/qtox-1.7.0-r1.ebuild]: I've been told by @kensington that there's usually no need for this and in fact the metadata.xml of qtcore also says the same thing: `Must only be used by packages that are known to use private parts of the Qt API.` If there anything to justify this?

File: net-im/qtox/qtox-1.7.0-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2016 Gentoo Foundation
```
Comment (gktrk) [inline net-im/qtox/qtox-1.7.0-r1.ebuild]: Please try to use `repoman commit` for committing changes, which not only checks for QA problems but also fixes some such as the outdated copyright year.

File: net-im/qtox/qtox-1.7.0-r1.ebuild (hunk 1 above)
Comment (soredake) [inline net-im/qtox/qtox-1.7.0-r1.ebuild]: I put the = on dev-qt/qtcore just to trigger rebuild when qt slot is changed. I'll retest today to see what component will show in error like this `qtox: /usr/lib64/libQt5Gui.so.5: version `Qt_5.7' not found (required by qtox)` and update the commit with = on correct dependency.

Comment (kensington) [inline net-im/qtox/qtox-1.7.0-r1.ebuild]: I've had a quick look in the source, and while there are a couple of ``QT_VERSION`` ifdefs, they all involve versions lower than our lowest version in the Gentoo tree.

Regarding the ``version 'Qt_5.7' not found`` error, I wonder if you have mixed Qt versions installed, or downgraded from 5.7 to 5.6?

Comment (soredake) [inline net-im/qtox/qtox-1.7.0-r1.ebuild]: >or downgraded from 5.7 to 5.6?

This.

**Comment by gktrk** [comment]
Ping. Is this still a valid PR?

**Comment by soredake** [comment]
I'll reopen if i catch this error again

**Pull request closed without merging**